### PR TITLE
chore(bench): add ciphertexts sum to integer benchmarks

### PR DESCRIPTION
**Pull request by soonum**
closes: _please link all relevant issues_

### Check-list:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Relevant issues are marked as resolved/closed, related issues are linked in the description
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci integer_full_bench

File: tfhe/benches/integer/bench.rs [hunk 1]
```diff
@@ -566,6 +566,68 @@ fn if_then_else_parallelized(c: &mut Criterion) {
...
+            let bench_id = format!("{bench_name}_{len}_ctxts::{param_name}::{bit_size}_bits");
+            bench_group.bench_function(&bench_id, |b| {
+                let (cks, sks) = KEY_CACHE.get_from_params(param, IntegerKeyKind::Radix);
+
+                let nb_ctxt = bit_size.div_ceil(param.message_modulus().0.ilog2() as usize);
+                let cks = RadixClientKey::from((cks, nb_ctxt));
+                // let modulus = cks.parameters().message_modulus().0.pow(nb_ctxt as u32) as u64;
+                //
+                // let max_value_for_bit_size = ScalarType::MAX >> (ScalarType::BITS as usize -
+                // bit_size);
```
Comment (tmontaigu) [inline tfhe/benches/integer/bench.rs:592]: remove comment out code

File: tfhe/benches/integer/bench.rs [hunk 2]
```diff
@@ -566,6 +566,68 @@ fn if_then_else_parallelized(c: &mut Criterion) {
...
+
+                let encrypt_values = || {
+                    let clears = (0..len)
+                        .map(|_| div_scalar(&mut rng, bit_size))
```
Comment (tmontaigu) [inline tfhe/benches/integer/bench.rs:596]: should probably use `gen_random_u256(&mut rng);` insead of `div_scalar(...)`

**Review by tmontaigu — APPROVED** [review]

**Comment by IceTDrinker** [comment]
@slab-ci cpu_signed_integer_test

**Pull request merged**